www/src/components/shell/CloudShell.js

```diff
@@ -100,6 +101,7 @@ function CloudShell() {
       childIsReady={ready}
     >
       <CreateRepositoryCard data={data} />
+      <OnboardingFooter />
```

Comment:
I skipped it for cloud shell as I initially thought that onboarding will have different flow, where the side menu is not visible.

To avoid putting it in multiple places I'd move it from here and `OnboardingFlow.js` to `OnboardingWrapper.js` as it is being used in both places.